test/spec/ol/style/icon.test.js

```diff
@@ -91,33 +92,29 @@ describe('ol.style.Icon', function() {
         src: src
       });
       var clone2 = original2.clone();
-      expect(original2.getSrc()).to.be(clone2.getSrc());
+      expect(clone2.iconImage_.getImageState() === ol.ImageState.LOADING);
```

Comment:
Interesting. I would expect the state to be the same as on the original. Is the original also still loading?

Reply:
Yes. It is loaded as soon as the browser gets some computation time.
But maybe it would be better to give a real http url here?

Reply:
or maybe better use `expect(clone2.iconImage_.getImageState()).to.eql(original2.iconImage_.getImageState());`